presto-kinesis/pom.xml

```diff
@@ -0,0 +1,321 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.facebook.presto</groupId>
+        <artifactId>presto-root</artifactId>
+        <version>0.107-SNAPSHOT</version>
+    </parent>
+
+
+
+  <!-- <groupId>com.facebook.presto</groupId>
+  <artifactId>presto-kinesis</artifactId>
+  <version>0.90-SNAPSHOT</version>
+  <name>presto-kinesis</name>
+  <url>http://maven.apache.org</url> -->
+
+  <artifactId>presto-kinesis</artifactId>
+    <description>Presto - Kinesis Connector</description>
+    <packaging>presto-plugin</packaging>
+
+  <properties>
+    <!-- <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding> -->
+    <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+  </properties>
+    
+  <dependencies>
+     <!-- <dependency>
+       <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>3.8.1</version>
+      <scope>test</scope>
+    </dependency>  -->
+
+    <dependency>
+        <groupId>org.weakref</groupId>
+        <artifactId>jmxutils</artifactId>
+        <scope>provided</scope>
+    </dependency>
+
+    <dependency>
```

Comment:
do not use `provided` scope for libraries which are not used as part of presto<->presto-connector interface.